src/elf/note.rs

```diff
@@ -49,10 +49,10 @@ pub const NT_PRSTATUS: u32 = 1;
 pub const NT_PRPSINFO: u32 = 3;
 
 ///Fields of siginfo_t.
-pub const NT_SIGINFO: u32 = 0x53494749;
+pub const NT_SIGINFO: u32 = 0x53_494_749;
```

Comment:
Hex literals are probably more readable in groups of 4 (same below):

```suggestion
pub const NT_SIGINFO: u32 = 0x5349_4749;
```

Reply:
@jan-auer Agreed, fixed